client/src/components/MovieCard/MovieCard.spec.js

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react'
+import MovieCard from './MovieCard'
+
+describe('MovieCard', () => {
+  it('renders', () => {
+    const title = 'Fight Club'
+    const year = '1999'
+    const genre = 'Drama'
+    render(
+      <MovieCard
+        title={title}
+        year={year}
+        genre={genre}
+        imgUrl="https://image.tmdb.org/t/p/w500/a26cQPRhJPX6GbWfQbvZdrrp9j9.jpg"
+      />
```

Comment:
I would suggest adding the props directly, as it helps to read it better - I think.
```suggestion
    render(
      <MovieCard
        title='Fight Club'
        year='1999'
        genre='Drama'
        imgUrl="https://image.tmdb.org/t/p/w500/a26cQPRhJPX6GbWfQbvZdrrp9j9.jpg"
      />
```

Reply:
dunno, I think both it viable. I would keep it like this, because I use the same data in different places